src/languages/en_GB.json

```diff
@@ -125,16 +125,60 @@
+      "2": "A <1>PICK</1> adds a usable civilisation for a player.",
+      "3": "A <1>GLOBAL PICK</1> or <3/> prevents either player from picking the civilisation again in a future turn.",
+      "4": "An <1>EXCLUSIVE PICK</1> or <3/> prevents only the picking player from picking the civilisation again in a future turn.",
+      "5": "A <1>NONEXCLUSIVE PICK</1> or <3/> does neither of those things, meaning players are still able to pick the civilisation again in a future turn.",
+      "6": "A <1>BAN</1> marks civilisation as not available for future turns.",
+      "7": "A <1>GLOBAL BAN</1> or <3/> prevents either player from picking the civilisation in a future turn.",
+      "8": "An <1>EXCLUSIVE BAN</1> or <3/> prevents only the opponent from picking the civilisation in a future turn and the player from banning the civilisation again in a future turn.",
+      "9": "A <1>NONEXCLUSIVE BAN</1> or <3/> prevents only the opponent from picking the civilisation in a future turn, and the player may ban the the civilisation again in a future turn, which makes no sense, so you probably want to use <5>EXCLUSIVE</5> instead anyway.",
+      "10": "A <1/> allows a player to remove a civilisation again which the opponent has picked in a previous turn. The <3>GLOBAL</3>, <5>EXCLUSIVE</5> and <7>NONEXCLUSIVE</7> modifiers have no meaning here."
+    },
+    "adminTurns": {
+      "title": "Admin Turns",
+      "1": "Admin turns or <1/> reveals the previous set of actions which were hidden",
+      "2": "The <1>REVEAL_PICKS</1>, <3>REVEAL_BANS</3> and <5>REVEAL_SNIPES</5> turns reveal the respective hidden actions that were executed before",
+      "3": "<0>REVEAL_ALL</0> reveals all hidden actions that were executed before."
+    },
+    "presetEditor": {
+      "title": "Preset Editor",
+      "1": "Presets are made up by individual Turns.",
+      "2": "Each turn has an acting player: The Host, the Guest, or neither of those, (so called \"Admin\" turns).",
+      "3": "You can add a Host/Admin/Guest turn by clicking the <1>+ New</1> button below the respective column.",
+      "4": "You can remove a turn by clicking the <1/> button to the right of the turn."
+    },
+    "presetModifiers": {
+      "title": "Modifiers",
+      "1": "If you want the players to execute two of their turns in parallel, check the <1>PARALLEL</1> checkbox of the <3>first</3> of those two turns (but do not check it for the second turn).",
+      "2": "Check the <1>HIDDEN</1> checkbox to hide the civilisation selected during that turn from the opponent and from spectators until it is revealed by an admin turn (see below)."
+    },
+    "draftModifiers": {
+      "title": "Modifiers",
+      "1": "A <1>PARALLEL</1> modifier means both the players to execute their turns in parallel. The picks or bans are revealed to the other player and spectators only after both the players have made their choices.",
```

Comment:
>The picks or bans are revealed to the other player and spectators only after both the players have made their choices.

Whoops, that's a lie :upside_down_face:  
You need to mark them as hidden to achieve that.

Reply:
🙈